services/config_service.lua

```diff
@@ -1,6 +1,61 @@
+function join(t1, t2)
+  for _, v in pairs(t2) do
+    table.insert(t1, v)
+  end
+  return t1
+end
 
 if service.service == "sigfox" and service.action == "updated" then
-  -- local parameters = Config.getParameters({service = "sigfox"})
+  local currentTime = os.time(os.date("!*t"))
+  local timestamp = currentTime * 1000000
+  local isoTime = os.date("!%Y-%m-%dT%H:%M:%S+00:00", currentTime)
+  local parameters = Config.getParameters({service = "sigfox"})
+  local payloadConfigs = {}
+  local channels = {}
+  if parameters.parameters.callbacks ~= nil then
+    for k, v in pairs(parameters.parameters.callbacks) do
+      if v.payloadConfig ~= nil then
+        join(payloadConfigs, v.payloadConfig)
+      end
+    end
+  end
+  for k, v in pairs(payloadConfigs) do
+    local channelName = ""
+    local properties = {}
+    local nestJson = string.match(v.resource, "data_in%.%a+%.")
+    if nestJson ~= nil then
+      channelName = string.sub(v.resource, string.len(nestJson) + 1)
+      properties.data_type = "JSON"
+    elseif string.match(v.resource, "data_in%.") ~= nil then
+      channelName = string.sub(v.resource, 9)
+      properties.data_type = "NUMBER"
```

Comment:
Is it always numbers? is there no boolean and string cases?

Reply:
The idea is created the basic config_io for Exosense. The detail setting needs to modify from Exosense UI.

Reply:
Right but if the value from Sigfox is a boolean or string then the resource type will be number?